protocols/exmp-sampling.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
This document needs very clear attribution text. For example:

*This protocol, including all text and photos, was initially created by ...*

You should put this in boldface at the bottom of the document.